src/gui/qgsattributeeditorcontext.h

```diff
@@ -132,6 +133,21 @@ class GUI_EXPORT QgsAttributeEditorContext
      */
     inline QgsMapCanvas *mapCanvas() const { return mMapCanvas; }
 
+    /**
+     * Sets the associated CAD dock widget, \a cadDockWidget, (e.g. to be used in map tools).
+     * \note Unstable API. This method is unstable API and may be modified or removed at any time.
+     * \see cadDockWidget()
+     * \since QGIS 3.10
+     */
+    void setCadDockWidget( QgsAdvancedDigitizingDockWidget *cadDockWidget );
+
+    /**
+     * Returns the associated map canvas (e.g. to zoom to related features).
```

Comment:
Can you fix this docstring and add a unstable api note too?

Reply:
Done